pkg/tnf/handlers/podsets/podsets.go

```diff
@@ -35,9 +35,9 @@ type PodSet struct {
 	Replicas    int
 	Ready       int
 	UpToDate    int
-	Available   int // this param exist in deployment and not exsit on statefulset
+	Available   int // this param exist in deployment and not exist on statefulset
```

Comment:
Shouldn't it be changed to "this param **exists** in deployment _but_ **does not exist** on statefulset" to be more correct in english? Anyway, I prefer the "apply" verb here: "this param **applies** only to deployments, not to statefulsets", because the param will always exist in this struct, but the value will be always 0 for statefulsets, right?

Reply:
Agreed.  I will adjust the strings.